openedx/core/djangoapps/theming/core.py

```diff
@@ -47,7 +47,7 @@ def comprehensive_theme_changes(theme_dir):
 
     favicon = component_dir / "static" / "images" / "favicon.ico"
     if favicon.isfile():
-        changes['settings']['FAVICON_PATH'] = str(favicon)
+        changes['settings']['FAVICON_PATH'] = "images/favicon.ico"
```

Comment:
@saleem-latif I think overriding `FAVICON_PATH` setting in comprehensive theming does not make sense at all since we already have `settings.FAVICON_PATH'] = "images/favicon.ico"` in env/common.py. Themed directory has precedence over default static files path so it would look for `images/favicon.ico` in themed directory first before falling back to default static files directory. I think we should remove this favicon logic here.